src/graphql/error-map.ts

```diff
@@ -106,6 +107,10 @@ export const mapError = (error: ApplicationError): CustomApolloError => {
         "Too many onchain addresses creation, please wait for a while and try again."
       return new TooManyRequestError({ message, logger: baseLogger })
 
+    case "CouldNotFindPhoneCodeError":
+      message = "Invalid or incorrect phone code entered."
```

Comment:
do we really need to override the message? maybe we could just set a default message of `PhoneCodeError` as this one?